docs/src/blueprints.md

```diff
@@ -0,0 +1,166 @@
+# Infragraph Blueprints
+
+Infragraph provides users with multiple **blueprints** that help define the foundational components of an infrastructure. These blueprints cover both **devices** and **fabrics**, enabling flexible and extensible modeling of network and compute infrastructure.
+
+Each blueprint is implemented as Python source code, allowing users to create classes and objects that inherit from the generated SDK. This approach makes it possible to define more realistic and customizable representations of devices and fabrics while maintaining compatibility with Infragraph’s schema and data models.
+
+All available blueprints can be found in the following directory:
+
+```
+src/infragraph/blueprints
+```
+
+---
+
+## Device Blueprints
+
+Device blueprints enable you to instantiate specific device types using the Infragraph schema format. Each blueprint encapsulates attributes, component definitions, and configuration logic that represent actual hardware or node types within a network topology.
+
+All device blueprints are organized under:
```

Comment:
Winston would like to see device blueprints organized under an additional level, by vendor - so dgx and cx5 would go under nvidia